### PR TITLE
revise index and edit views for admin paints

### PR DESCRIPTION
**Pull request by Robertgervais**

### CODE REVIEW

**Comment by Robertgervais** [comment]
paintsedit1
paintsedit2
paintsedit3
paintsindex

**Comment by Robertgervais** [comment]
Would you like for me to revise /admin/bikes/admin_bikes_table right now or later?

File: app/views/admin/paints/edit.html.haml [hunk 1]
```diff
@@ -1,57 +1,46 @@
...
-      .control-group.chosen-select
-        = f.label :color_id, class: "control-label"
-        .controls
-          = collection_select( :paint, :color_id, Color.all, :id, :name, prompt: "Choose color" )
-      .control-group.chosen-select
-        = f.label :secondary_color_id, class: "control-label"
-        .controls
-          = collection_select( :paint, :secondary_color_id, Color.all, :id, :name, prompt: "Choose color" )
-      .control-group.chosen-select
+= form_for [:admin, @paint], html: { class: "form form-horizontal span6" } do |f|
```
Comment (sethherr) [inline app/views/admin/paints/edit.html.haml]: I don't think this form needs these classes?

File: app/views/admin/paints/index.html.haml [hunk 2]
```diff
@@ -45,24 +46,24 @@
                   Yesterday at #{paint.created_at.strftime("%-I %p")}
```
Comment (sethherr) [inline app/views/admin/paints/index.html.haml]: Looks like a custom date function...

**Pull request merged**